css/stylesheet.css

```diff
@@ -0,0 +1,803 @@
+}
+.icons {
+  margin: auto;
+  margin-bottom: 10%;
+  width: 100%;
+  text-align: center;
+}
+.icons span{
+  color: white;
+  display: inline-block;
+  -webkit-transition: -webkit-transform .25s ease-in-out;
+  transition: transform .25s ease-in-out;
+  line-height: 1;
+}
+.icons span:hover {
+  border: white;
+  -ms-transform: scale(1.25,1.25);
+  -webkit-transform: scale(1.25,1.25);
+  transform: scale(1.25,1.25);
+  transition: .25s ease-in-out;
+}
+.icon-menu:before {
+   content: "\e608";
+}
+
+/*Nav bar mobile*/
+.icon-menu {
+  width:100%;
+  text-align:center;
+}
+
+.mobile-menu ul {
+  list-style-type:none;
+  margin: 0 auto;
+  padding-left:0;
+  text-align:center;
+  width:97.2%;
+  position: absolute;
+  background: #999f9f;
+}
+
+.mobile-menu li a {
+  display:block;
+  font-family: Helvetica;
+  color: #fff;
+  text-transform:uppercase;
+  text-decoration: none;
+  font-size: 1.3em;
+  line-height: 2em;
+  padding: 10px 0;
+}
+
+.mobile-menu input[type=checkbox]{
+    display: none;
+}
+
+.mobile-menu input[type=checkbox]:checked ~ .menu{
+  display: block;
+}
+
+.mobile-menu ul {
+  display: none;
+  z-index: 100;
+}
+
+.icon-menu {
+    display:inline-block;
+}
+/*nav bar end*/
+
+a {
+  text-decoration: none;
+}
+footer {
+  text-align: center;
+}
+.p-home {
+  background-image: url("https://i.imgur.com/oVRIX4Z.png");
+}
+.port-navbar {
+  display:flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-content: flex-start;
+  align-items: center;
+  margin-left: 30%;
+  margin-right: 30%;
+}
+.port-navbar a {
+  display: inline-block;
+  text-align: center;
+  font-size: 1em;
+  font-family: "Droid sans", sans-serif;
+  text-decoration: none;
+  color:#a3a3a3;
+}
+.port-navbar a:hover {
+  background-color: red;
+}
+.icon-menu {
+    display:inline-block;
+}
+nav ul, nav:active ul {
+  display: none;
+}
+nav li {
+  text-decoration: none;
+  text-align: left;
+  width: 100%;
+  padding: 10px 0;
+}
+nav:hover ul {
+   display: block;
+}
+
+.protfolio_pieces{
+  display:flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: flex-start;
+  margin-bottom: 4em;
+  text-align: center;
+}
+.l-icon-outer {
+  position: relative;
+  margin: auto;
+  border-radius: 50%;
+  height: 75px;
+  width: 75px;
+  background-color: #fc6432;
+}
+.l-droplet-icon {
+  position: absolute;
+  left: 25px;
+  top: 25px;
+  margin: auto;
+  width: 25px;
+  height: 25px;
+  overflow: hidden;
+  border-bottom-right-radius: 50%;
+  border-bottom-left-radius: 50%;
+  border-top-left-radius: 50%;
+  -webkit-transform:rotate(-45deg);
+  z-index: 99;
+  background: white;
+}
+.l-droplet-icon-interior {
+  position: absolute;
+  left: 2.5px;
+  top: 2.5px;
+  margin: auto;
+  width: 20px;
+  height: 20px;
+  overflow: hidden;
+  border-bottom-right-radius: 50%;
+  border-bottom-left-radius: 50%;
+  border-top-left-radius: 50%;
+  z-index: 150;
+  background: #fc6432;
+}
+ul {
```

Comment:
I would not put this selector here. This is a base style or something that should be in the reset.